### PR TITLE
Add error_type and error_code columns to system.runtime.queries

### PR DESCRIPTION
**Pull request by kokosing**
Add error_type and error_code columns to system.runtime.queries

### CODE REVIEW

File: presto-main/src/main/java/io/prestosql/connector/system/QuerySystemTable.java [hunk 1]
```diff
@@ -55,6 +56,8 @@
     public static final ConnectorTableMetadata QUERY_TABLE = tableMetadataBuilder(QUERY_TABLE_NAME)
             .column("query_id", createUnboundedVarcharType())
             .column("state", createUnboundedVarcharType())
+            .column("error_type", createUnboundedVarcharType())
```
Comment (sopel39) [inline presto-main/src/main/java/io/prestosql/connector/system/QuerySystemTable.java]: I wonder if new columns shouldn't be added as last to not break existing system runtime queries. On the other hand, if someone depends on column index, that's probably a bad practice

Comment (kokosing) [reply presto-main/src/main/java/io/prestosql/connector/system/QuerySystemTable.java]: I think I can move it anyway. It does matter to me.

File: presto-main/src/main/java/io/prestosql/connector/system/QuerySystemTable.java [hunk 2]
```diff
@@ -116,7 +120,10 @@ public RecordCursor cursor(ConnectorTransactionHandle transactionHandle, Connect
                     toTimeStamp(queryStats.getCreateTime()),
                     toTimeStamp(queryStats.getExecutionStartTime()),
                     toTimeStamp(queryStats.getLastHeartbeat()),
-                    toTimeStamp(queryStats.getEndTime()));
+                    toTimeStamp(queryStats.getEndTime()),
+
+                    Optional.ofNullable(queryInfo.getErrorType()).map(Enum::name).orElse(null),
+                    Optional.ofNullable(queryInfo.getErrorCode()).map(ErrorCode::getCode).orElse(null));
```
Comment (findepi) [inline presto-main/src/main/java/io/prestosql/connector/system/QuerySystemTable.java]: i think `ErrorCode::getName` would be more useful

(i'd leave the column name as is)

**Comment by kokosing** [comment]
Comments addressed

**Pull request merged**